lib/min_heap.rb

```diff
@@ -8,26 +8,32 @@ def initialize(key, value)
 end
 
 class MinHeap
-
   def initialize
     @store = []
   end
 
   # This method adds a HeapNode instance to the heap
-  # Time Complexity: ?
-  # Space Complexity: ?
+  # Time Complexity: O(log n)
+  # Space Complexity: O(1)
```

Comment:
Since `heap_up` is recursive, you have to account for space used on the call stack.  So this is O(log n) space complexity.